### ISSUE TITLE
make c2c compiler available for all supported platforms.

### ISSUE THREAD

**Issue by samadadi** [c2lang/c2compiler#31]
please make c2c compiler available for all supported platforms in **release** section including **windows** as zip or installer file. I don't use c2 programming language because of compiler installation process.

**Comment by luciusmagn** [comment]
Hello, Windows is a difficult thing. Compiling your own LLVM & Clang is an arduous, yet necessary process, twice as much on Windows. Bas doesn't use Windows at all, I use it marginally.

I was a pioneer in usage on Windows back in 2015 and found that the best solution is to use the Cygwin compatibility layer. I ended up making some instructions that are in INSTALL_WIN.md, but due to these limitations (c2c simply cannot work Windows natively without major changes to both the modified Clang and C2C itself), it is impossible to just make a simple installer or a reliable release.

The fact that C2 doesn't have a release cycle and versioning is also an obstacle.

**TL;DR** C2C is available for all officially supported platforms. Windows is not an officially supported platform. If you want to use C2C on Windows, please refer to INSTALL_WIN.md.
